.npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
Had to run this because otherwise `npm install` spits out something like the following:

```sh
npm ERR! code ERESOLVE
npm ERR! ERESOLVE could not resolve
npm ERR! 
npm ERR! While resolving: @remix-run/eslint-config@2.4.1
npm ERR! Found: react@18.3.0-canary-c5b937576-20231219
npm ERR! node_modules/react
npm ERR!   react@"^18.3.0-canary-c5b937576-20231219" from the root project
npm ERR!   peer react@"18.3.0-canary-c5b937576-20231219" from react-dom@18.3.0-canary-c5b937576-20231219
npm ERR!   node_modules/react-dom
npm ERR!     react-dom@"^18.3.0-canary-c5b937576-20231219" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.0.0" from @remix-run/eslint-config@2.4.1
npm ERR! node_modules/@remix-run/eslint-config
npm ERR!   dev @remix-run/eslint-config@"2.4.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: react@18.2.0
npm ERR! node_modules/react
npm ERR!   peer react@"^18.0.0" from @remix-run/eslint-config@2.4.1
npm ERR!   node_modules/@remix-run/eslint-config
npm ERR!     dev @remix-run/eslint-config@"2.4.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/brookslybrand/.npm/_logs/2024-01-16T17_56_25_942Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/brookslybrand/.npm/_logs/2024-01-16T17_56_25_942Z-debug-0.log
```

We can also do `force=true` instead, but still spits out all the above just with some warnings. To me this just seems like something worth taking off the plate of someone trying to get this running quickly.